composer.json

```diff
@@ -24,47 +24,53 @@
             "email": "joe@alt-three.com"
         }
     ],
+    "replace": {
+        "paragonie/random_compat": "*",
```

Comment:
Since Cachet requires PHP 7.1, we can prevent the installation of this package by saying we "replace" it. Similar for the mbstring polyfill, since we require the actual extension.